types/react-router/test/InheritingRoute.tsx

```diff
@@ -13,6 +13,9 @@ export default class CustomRoute extends Route<CustomRouteInterface> {
     // react-fiber's render function can also return an array of elements,
     // but in this case it's assumed that a JSX.Element is returned.
     const maybeElement = super.render() as JSX.Element;
+    const aProperty: string = this.props.aProperty;
```

Comment:
adding this here so it actually catches that the inheritance is working as expected.
(well, this tests are from 2017 and I quite doubt many people follow this pattern 🙈 but wanted to make it as backards-compatible as possible 😄🤞)